### DISCUSSION TITLE
Deployer receipts for Neos and Uberspace

### DISCUSSION THREAD

**Discussion by jonnitto** [deployphp/deployer#2343] [General]
I've created a small package to interact with [Neos CMS](https://www.neos.io) and [Uberspace](https://uberspace.de)

[![Social Preview](https://user-images.githubusercontent.com/4510166/101900466-d099fb80-3baf-11eb-9599-e5c721001736.png)](https://github.com/jonnitto/uberspace-deployer#readme)

https://github.com/jonnitto/uberspace-deployer#readme

**Comment by antonmedv** [comment]
Awesome! Based on v7 Deployer?

**Reply by jonnitto** [reply]
Not yet, but will come soon. Neos 7 (which will be released these days) will have support for symfony 5 ([As I asked here](https://github.com/deployphp/deployer/discussions/2342))
